.env.example

```diff
@@ -0,0 +1,2 @@
+DARK_SKY_API= 'api_key_for_dark_sky_goes_here'
```

Comment:
This is nice for reference if someone wants to clone this down, could point to this file in the README